content/documentation/Ox64/Software/Flashing.adoc

```diff
@@ -57,71 +137,44 @@ Verify that your copy of `BouffaloLabDevCube-v1.8.3.zip` matches the hashes belo
 * SHA1: `0f2619e87d946f936f63ae97b0efd674357b1166`
 * SHA256: `e6e6db316359da40d29971a1889d41c9e97d5b1ff1a8636e9e6960b6ff960913`
 
-== Optional: create a combined SoC image
-
-Use the following commands to combine `m0_lowload_bl808_m0.bin`, `d0_lowload_bl808_d0.bin`, and `bl808-firmware.bin` into a single image. This is mainly useful for troubleshooting (e. g. when using DevCube v1.8.4 or later).
-
-----
-cd ~/ox64/openbouffalo
-
-fallocate -l 0x800000 bl808-combined.bin
-dd conv=notrunc if=m0_lowload_bl808_m0.bin of=bl808-combined.bin
-dd conv=notrunc if=d0_lowload_bl808_d0.bin of=bl808-combined.bin seek=$((0x100000))B
-cat bl808-firmware.bin >> bl808-combined.bin
-----
-
-== Establish serial communication from PC to Ox64 using Pi Pico
-
-Put the Raspberry Pi Pico board into programming mode.
-
-* Press the BootSel button
-* Apply power by plugging the USB cable to PC
-* Release the BootSel button
+== Flashing the Ox64
 
-NOTE: As an alternative to pressing the BootSel button, you can also connect the probe point `TP6` (located on the bottom of the Pico board) to any ground point (e.g. pin 28).
+Put the Ox64 into programming mode:
 
-Copy `picoprobe.uf2` file into the new device `/media/<user>/RPI-RP2`.
+* Press the BOOT button
+* Apply power or re-plug the USB cable
+* Release the BOOT button
 
- cp ~/ox64/pico/picoprobe.uf2 /media/<user>/RPI-RP2
+=== CLI flashing method
 
-After flashing, verify that you have two serial interfaces (when the Pico is connected to your PC):
+Each time you open a new terminal window you will need to reactivate the virtual environment.
 
-----
-$ ls /dev/ttyACM*
+ `. ~/ox64_venv/bin/activate`
```

Comment:
I don't think we need to mention this a second time. Especially for those using alternative means of managing PIP packages (e.g. `conda`).

Reply:
I agree.